src/Automation/Interfaces/ITargetElementLocator.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.Core.Bases;
+
+namespace Axe.Windows.Automation
+{
+    internal interface ITargetElementLocator
+    {
+        A11yElement LocateRootElement(int processId);
```

Comment:
>A11yElement [](start = 8, length = 11)

Can this be ICoreA11yElement?

Reply:
I would like that as well, but I don't have time to make that change if _anything_ goes awry. I aslo think it might be more appropriate when we try to remove A11yElement out of AI-Win altogether.